Installing-and-Configuring/OS/index.md

```diff
@@ -58,17 +58,17 @@ Your other partitions may be encrypted or not based on your preferences and this
 
 ### Using TPM for decryption of drives
 
-Heads can seal a Disk Unlock Key using the TPM.  This will ensure that upon booting only a disk unlock passphrase and the proper PCR state can access the data on the root partition.  This approach uses a Security Dongle for disk decryption and may be used to lock the encrypted data to the hardware.
+Heads can protect a LUKS Disk Unlock Key using the TPM.  This will ensure that upon booting with Heads only a TPM disk unlock passphrase and the proper PCR state can access the data on the root partition.  In this model the TPM enforces rate limits to prevent brute force attacks.  If the disk is moved to a different system the [LUKS disk unlock passphrase/recovery key](/Keys/#tpm-disk-encryption-key) in slot 0 may be used to access the encrypted data.  This model is similar to the [Security Dongle](#security-dongle-with-luks-root) model.
 
-If this feature is enabled, heads will ask for a Disk Unlock Passphrase when saving the default boot partition configuration.  This is not the same as the passphrase/key used by LUKS.  The Disk Unlock passphrase will be used to generate a Disk Unlock Key to be placed in the second LUKS keyslot for the root partition.  The disk unlock key is sealed in the TPM along with measurements taken at that time.
+If the Heads TPM drive decryption feature is enabled, Heads will [ask for](/Keys/#tpm-disk-encryption-key) a Disk Unlock Passphrase (which might be a numerical PIN) when saving the default boot partition configuration.  This is not the same as the passphrase/key used by LUKS.  The TPM Disk Unlock passphrase will be used to generate a LUKS Disk Unlock Key to be placed in keyslot 1 for the root partition.  The LUKS disk unlock key is sealed in the TPM along with measurements taken at that time.
```

Comment:
If the Heads [TPM drive decryption feature is not disabled](https://github.com/osresearch/heads/blob/c3b0bd6ffbe816430dd41ef54e649af52ed1ff3b/boards/librem_13v2/librem_13v2.config#L31), Heads will [ask for a Disk Unlock Key passphrase](/Keys/#tpm-disk-encryption-key) (which might be a PIN: A short passphrase/password) when saving the default boot partition configuration. This is not the same as the passphrase used by LUKS. The TPM Disk Unlock passphrase will be used to generate a LUKS Disk Unlock Key to be placed in LUKS key slot 1 for the root partition. The LUKS Disk Unlock Key is sealed in the TPM along with measurements (LUKS header, firmware measurement, Disk Unlock Key passphrase unlocking TPM NV reserved memory space) taken at that time.

Reply:
updated.  push in a minute